docs/_docs/01_getting-started/install.md

```diff
@@ -11,7 +11,12 @@ nav_title_link: /docs/
 
 ## Install Terragrunt
 
-Note that third-party Terragrunt packages may not be updated with the latest version, but are often close. Please check your version against the latest available on the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases).
+### Download from releases page
+
+You can install Terragrunt manually by going to the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases), downloading the binary for your OS, renaming it to `terragrunt`, and adding it to your PATH.
+
```

Comment:
Just realized these steps are missing useful details! Here's a slightly better, updated version:

```suggestion
1. Go to the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases).
1. Downloading the binary for your operating system: e.g., if you're on a Mac, download `terragrunt_darwin_amd64`; if you're on Windows, download `terragrunt_windows_amd64.exe`, etc.
1. Rename the downloaded file to `terragrunt`.
1. Add execute permissions to the binary. E.g., On Linux and Mac: `chmod u+x terragrunt`.
1. Put the binary somewhere on your `PATH`. E.g., On Linux and Mac: `mv terragrunt /usr/local/bin/terragrunt`.
```

Reply:
I have added the missing details.